antora-ui-camel/src/partials/header-content.hbs

```diff
@@ -16,7 +16,10 @@
                 </div>
                 </div>
                 {{else}}
-                <a class="navbar-item navbar-topics" href="{{url}}">{{name}}</a>
+                <div class="navbar-item-section">
+                  <img src="{{siteRootPath}}/_/img/{{name}}.svg">
+                  <a class="navbar-item navbar-topics" href="{{url}}">{{name}}</a>
+                </div>
```

Comment:
Why not make this an anchor tag with an image within it, instead of div with image and anchor. Would make the whole item clickable.